.github/workflows/swift.yaml

```diff
@@ -0,0 +1,41 @@
+name: Swift - Workflow - Cocoapods (Xcode 11.2.1)
+
+on: [push]
+
+jobs:
+  bundle:
+    runs-on: macOS-latest
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Install Bundler
+      run: |
+        gem update --system
+        gem install bundler
+    - name: Bundle Install
+      run: bundle install
+  
+  build_docs:
+    runs-on: macOS-latest
+    needs: bundle
+    
+    steps:
+    - name: Documentation
+      run: |
+        brew install sourcedocs
+        .buildscript/build_swift_docs.sh /tmp/swiftdocs
```

Comment:
Does this job serve a purpose? Looks like this output isn't used at all. I'm planning to make a separate workflow to build and validate the entire doc site.

Reply:
That makes sense. I'll skip it from this one then.

Reply:
Oh, never mind – i just read our travis config again and realized this was there to validate that the build actually passed. I guess we broke it at one point. We should leave this in then (but also copy the comment back over).